include/dca/phys/parameters/mci_parameters.hpp

```diff
@@ -86,7 +96,9 @@ class MciParameters {
   int measurements_;
   int walkers_;
   int accumulators_;
+  bool shared_walk_and_accumulation_thread_;
   bool adjust_self_energy_for_double_counting_;
+  int error_computation_type_;
```

Comment:
why is this an int and not an ErrorComputationType

Reply:
I would need to add specializations to MPITypeMap for each enumeration. Do you think it is a better idea or would there be a way to specialize the class in a way to match all possible enumerations?